target/linux/mvebu/config-6.1

```diff
@@ -1,6 +1,7 @@
 CONFIG_AHCI_MVEBU=y
 CONFIG_ALIGNMENT_TRAP=y
 CONFIG_ARCH_32BIT_OFF_T=y
+CONFIG_ARCH_FORCE_MAX_ORDER=11
```

Comment:
`CONFIG_FORCE_MAX_ZONEORDER=11` was renamed into `CONFIG_ARCH_FORCE_MAX_ORDER=11`, this should go into `target/linux/generic/config-6.1` IMO

Reply:
Agree.
But this affects all (17) current 6.1 configs and not just mvebu, so maybe a separate pre/post change is more appropriate here.

Reply:
Referenced changes have been merged in b308bd50efaadd2fdfa58123b0e25896478c882b, so this is no longer an issue. Dropped the corresponding commit from this PR as the flag is now defined in _generic_ and we don't need any overrides for _mvebu_.